src/components/users/users.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import SingleUser from '../singleUser/singleUser';
+
+const Users = ({users}) => {
```

Comment:
this one is probably not needed at all, as you can just have `<UsersBalance />` component